### ISSUE TITLE
Caixas no editor atividade, para o professor e resposta juntos

### ISSUE THREAD

**Issue by simas0** [livro-aberto/BookCloud#233]
Ao se clicar em Atividade na barra de ferramentas do editor deveria aparecer todo o texto abaixo. Atenção à alteração no rótulo da atividade.

```
.. _ativ-cap-titulo-da-atividade:

Atividade: título da atividade
------------------------------

.. admonition:: Para o professor

   **Objetivos específicos:**
   
   **Recomendações para o desenvolvimento da atividade:**

Texto da atividade

.. admonition:: Resposta 

   texto da resposta

```

**Comment by simas0** [comment]
A solução certa é que Atividade não seja uma seção, mas um ambiente. Observe que atualmente, incluindo a solução acima, o ambiente pode ser fechado com o fechamento do ambiente resposta.

**Comment by diogor** [comment]
O rótulo então ficaria: _ativ-<nome>, já incluindo o para o professor e resposta?
Como entrada então teríamos 5:
- Título
- Objetivos específicos
- Recomendações e sugestões
- Texto
- Texto da resposta

Vou adicionar isso à branch feature/editor, e fecho junto até segunda.

**Comment by simas0** [comment]
Sim, mas o rótulo de fazer referência ao nome do capítulo também, ou outra maneira de evitar ambiguidades. Do contrário pode haver ambiguidade nestes rótulos.
